test/multiverse/suites/sidekiq/Envfile

```diff
@@ -9,20 +9,14 @@ end
 SIDEKIQ_VERSIONS = [
   [nil, 2.7],
   ['6.4.0', 2.5],
-  ['5.0.3', 2.3],
-  ['4.2.0', 2.2]
+  ['5.0.3', 2.4],
+  ['4.2.0', 2.4]
 ]
 
 def gem_list(sidekiq_version = nil)
-  # must use older gems for older Rubies
-  gem_redis = RUBY_VERSION <= "2.3.0" ? "gem 'redis', '<= 4.1.0'" : ""
-  gem_connection_pool = RUBY_VERSION <= "2.3.0" ? "gem 'connection_pool', '<= 2.2.2'" : ""
-  <<-RB
-    gem 'rack', "~> #{RUBY_VERSION >= '2.3.0' ? '2.2.4' : '2.1.4.1'}"
+  <<~RB
+    gem 'rack', "~> 2.1.4.1"
```

Comment:
I think this should 2.2.4 based on the 
```
gem 'rack', "~> #{RUBY_VERSION >= '2.3.0' ? '2.2.4' : '2.1.4.1'}"
```

Reply:
YIKES. I wish this would've complained lol. [Fixed](https://github.com/newrelic/newrelic-ruby-agent/pull/1778/commits/10d7ca76aac24e3ea2a566fc48164d674342eb70#diff-ad4c384c68c03576217a4504b324e5e6dd33d51d6188557d25686f280def707bR17-R18).

Reply:
Weird, if it didn't even break it i wonder why we're doing that in the first place! such mystery